README.md

```diff
@@ -4,21 +4,21 @@
 
 --------------
 
-
-* [Overview](#overview)
-* [Downloads](#downloads)
-* [Changelog](#changelog)
-* [Compilation](#compilation)
-* [FFmpeg vs. Libav](#ffmpeg-vs-libav)
-* [Release cycle](#release-cycle)
-* [Bug reports](#bug-reports)
-* [Contributing](#contributing)
-* [Relation to MPlayer and mplayer2](#relation-to-mplayer-and-mplayer2)
+* [#Overview](#overview)
+* [#System requirements](#system-requirements)
+* [#Downloads](#downloads)
+* [#Changelog](#changelog)
+* [#Compilation](#compilation)
+* [#FFmpeg vs. Libav](#ffmpeg-vs-libav)
+* [#Release cycle](#release-cycle)
+* [#Bug reports](#bug-reports)
+* [#Contributing](#contributing)
+* [#Relation to MPlayer and mplayer2](#relation-to-mplayer-and-mplayer2)
+* [#Contact](#contact)
+* [#License](#license)
```

Comment:
Why all the "#"? These just make it look uglier here.

Reply:
So that you can tell that they are anchor links, ie. that they just take you to a specific part of the current page instead of to a different website.

Reply:
Why not just have an outline section and an external links section?